lib/std/heap.zig

```diff
@@ -524,7 +524,7 @@ pub fn StackFallbackAllocator(comptime size: usize) type {
 
         /// This function both fetches a `Allocator` interface to this
         /// allocator *and* resets the internal buffer allocator.
-        pub fn get(self: *Self) Allocator {
+        pub fn allocator(self: *Self) Allocator {
             self.fixed_buffer_allocator = FixedBufferAllocator.init(self.buffer[0..]);
```

Comment:
This does not actually seem to fix #16344, because the internal fixed_buffer_allocator is still getting reinitialized here when calling the `allocator()` function multiple times.

Reply:
I accidentally messed up my rebase, fixed now.